testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js

```diff
@@ -2797,6 +2816,46 @@ QUnit.module('datebox with time component', {
         }
     });
 
+    QUnit.test('date box should chenge behavior if adaptivityEnabled option is changed at runtime', function(assert) {
```

Comment:
I suggest splitting this test scenario into 2 tests:
- Change `adaptivityEnabled` from `false` to `true`
- Change `adaptivityEnabled` from `true` to `false`